CONTRIBUTING.md

```diff
@@ -6,13 +6,13 @@ possible.
 ... (in particular how this is synced with internal changes to the project)
 
 ## Pull Requests
-We actively welcome your pull requests.
```

Comment:
As far as I can tell, adding new lines adds a bigger interval between the ordered list items. IMO the rendered markdown looks better with the trailing double whitespace. Would you rather have the extra newline instead?